ceph-releases/ALL/rhel7/daemon-base/__DOCKERFILE_PREINSTALL__

```diff
@@ -18,3 +18,5 @@ LABEL description="Red Hat Ceph Storage 3"
 LABEL summary="Provides the latest Red Hat Ceph Storage 3 on RHEL 7 in a fully featured and supported base image."
 LABEL io.k8s.display-name="Red Hat Ceph Storage 3 on RHEL 7"
 LABEL io.openshift.tags="rhceph ceph"
+LABEL io.openshift.expose-services
```

Comment:
Unfortunately this introduced a docker built failure, and the error is "LABEL must have two arguments"

https://github.com/ceph/ceph-container/pull/1143 deletes this line.